modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -25,49 +25,63 @@ Use the search box to find the transmission based on criteria such as:
 * Filename
 * Control number
 * Endpoint
-* Date the file was received
-+
-Use the filters to find transmissions based on:
-+
+* Received date of the file +
+. You can then use the filters to find transmissions based on:
 * *Type* +
 Type of message transmission:
 ** *Both directions*
 ** *Sent to partners*
 ** *Received from partners*
-* *From Date*
-Filter for transmissions that sent or received after this date. 
+* *From Date* +
+Filter for transmissions sent or received after this date.
 * *To Date* +
-Filter for transmissions that are sent or received up to this date. 
+Filter for transmissions sent or received up to and including this date.
 * *Ack Status* +
 Filter for transmissions based on their acknowledgment status:
 ** *All*
 ** *Overdue*
-** *Rejected*
-. Select the transmission that matches your search criteria and review the summary information: 
+** *Rejected* +
+* *Message Processing Status* +
+Filter for transmissions based on their message processing status:
+** *All*
+** *Delivered*
+** *Errored*
+** *Processing*
+** *Received*
+* *Partner name* +
+Filter for transmissions based on your partner's name. +
+[NOTE]
```

Comment:
Does this have to be a notice, or can we just move it up to follow "...partner's name."?

Reply:
Done